src/plugins/intel_gna/tests/functional/shared_tests_instances/subgraph_tests/concat_multi_input.cpp

```diff
@@ -26,19 +26,20 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
     InferenceEngine::Precision::FP16,
 };
 
-std::map<std::string, std::string> additional_config = {
-    {"GNA_DEVICE_MODE", "GNA_SW_EXACT"},
-    {"GNA_COMPACT_MODE", "NO"},
-    {"GNA_SCALE_FACTOR_0", "2048"},
-    {"GNA_PRECISION", "I16"},
-};
+std::vector<std::map<std::string, std::string>> configs = {{
```

Comment:
is style formatting properly applied.

Reply:
Yes, if it wasn't, the code style CI task would fail, right?